app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -363,21 +372,25 @@ def handle_successful_status_update(enrollment, response)
       reason: 'Successful status update',
       job_name: self.class.name,
     )
-    enrollment.profile.activate_after_passing_in_person
     enrollment.update(
       status: :passed,
       proofed_at: proofed_at,
       status_check_completed_at: Time.zone.now,
     )
 
-    # send SMS and email
-    send_enrollment_status_sms_notification(enrollment: enrollment)
-    send_verified_email(enrollment.user, enrollment)
-    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_email_initiated(
-      **email_analytics_attributes(enrollment),
-      email_type: 'Success',
-      job_name: self.class.name,
-    )
+    unless fraud_pending?(enrollment)
+      enrollment.profile.activate_after_passing_in_person
+
+      # send SMS and email
+      # MW: Do we still want to send the SMS in the fraud case?
+      send_enrollment_status_sms_notification(enrollment: enrollment)
```

Comment:
This is SMS Notification of Proofing. My gut is that it should only be sent if the user isn't fraudulent, but I think it's potentially debatable.

Reply:
The proofing is still completing, correct? The SMS notification is so that the user becomes aware that the proofing job has completed and a determination has been made. It doesn't note anything about the status of the determination. I think we want to keep it in place.